sections/plate_notation.tex

```diff
@@ -72,7 +72,40 @@ \subsection{Author-Topic \gls{lda} and Category-Topic \gls{lda}}
 	\label{fig:metadata_lda}
 \end{figure*}
 
-\subsection{Pachinko}
-\todo[inline]{Description and plate notation}
+\subsection{Pachinko Allocation}
+In order to handle the hierarchical structure of the taxonomy metadata field, we will be using a hierarchical topic model, namely pachinko \citet{li2006pachinko}.
+Pachinko allocation generalizes \gls{lda}, to be able to operate on any DAG-structure.
 
+The DAG structure represents topics which in pachinko are distributions over words and/or other topics.
+It setup in a tree structure with the first layer being the root node, a topic which all documents are part of, and the bottom layer consisting of leaf nodes which are words. 
+
+For simplicity we will be using five-level pachinko tree structure, with each node in a layer having an edge to each node in the next layer.
+We also therefore limit the taxonomy chain to the first $3$ taxonomy entries, in order to fit with our DAG-structure.
+This DAG-structure is visualizated in \autoref{fig:pachinko_dag}
+
+For each word a chain of topics is sample by sampling one topic from each layer of the structure, and finally a word is chosen based on the last topic.
+More specifically, for each topic starting with the root topic, we sample a topic from the next layer $Z_{wl}$ from that node's topic distribution $\theta_l^(d)$ based on observed data from that document.
+
+\begin{figure}
+	\centering
+	\includegraphics[width= 0.5 \linewidth]{figures/pachinko_dag.png}
+	\caption{Illustration of the DAG-structure for five-level PAM}
+	\label{fig:pachinko_dag}
+\end{figure}
+
+\begin{figure}
+	\centering
+	\includegraphics[width= 0.9 \linewidth]{figures/five_layer_pachinko.png}
+	\caption{Plate notation for five-level PAM}
+	\label{fig:pachinko_plate}
+\end{figure}
+\todo[inline]{make pretty picture for pachinko plate notation}
+
+\subsubsection{Modification to Pachinko Allocation}
+Since only $25\%$ of the documents have an observed taxonomy, we can then lock the words in the observed documents to be in the corresponding topics within the pachinko DAG instead of conteniously sampling them using gibbs sampling.
```

Comment:
```suggestion
Since only $25\%$ of the documents have an observed taxonomy, we can then lock the words in the observed documents to be in the corresponding topics within the pachinko DAG instead of continuously sampling them using Gibbs sampling.
```